api/functionblock/v1alpha1/functionblock.proto

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright © 2021 Ci4Rail GmbH <engineering@ci4rail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package functionblock;
+option go_package = "functionblock/v1alpha1";
+
+// -------- Meta ------------
+message Context {
+  // A message identifying key for a command-response pairs, e.g. an UUID the
+  // clients sends on the request.
+  string value = 1;
+}
+
+// ------- Commands ---------
+message Command {
+  Context context = 1;
+  oneof type {
+    ConfigurationControl configurationControl = 2;
```

Comment:
Add "describe" command, so that the client can query properties of that function block (e.g. number of channels, capabilities)

Reply:
added